### DISCUSSION TITLE
Music Audio Not Playing after recording a video with streamlabs.

### DISCUSSION THREAD

**Discussion by bangagurlmusic** [ExistentialAudio/BlackHole#730] [Q&A]
Hello, So, I've been using Blackhole with streamlabs to record music production tutorials in logic pro.  I've literally made about 15 videos without issue.  However, today it appears that when I record a video in streamlabs the music from my logic pro project is not being captured in OBS.  I can only hear my vocals that are coming through Yeti.  I did not change any of my settings, so I am not sure why it just stop working.  I set my computer and logic pro settings to multi device output and input is my interface.  Any ideas as to what went wrong?

**Comment by devinroth** [comment]
Something must have changed. Could you share screenshots of your setup?

Devin

**Comment by bangagurlmusic** [comment]
So, basically I installed the blackhole 2 channel as well because the 16channel wasn't working.  I was able to get it to work with the 2 channel.  But, I'll send you both set ups.  However, when I swap out the 2 channel with the 16 channel via streamlabs and attempt to basically screen record my logic pro program for tutorial purposes.  I can hear the sound coming in through my scarlett interface.  But, when I stop recording and playback the video, I do not hear any audio.  I mean i guess i can move forward with the 2 channel, but I figure the 16 channel might be more useful for i guess more outputs?  Either way, I am curious as to why the 16 channel just stopped working.  Please see images for my setup.

<img width="652" alt="Screen Shot 2023-05-27 at 8 28 51 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/798ceb87-9460-499d-ad73-3e7a9ae63191">
<img width="389" alt="Screen Shot 2023-05-27 at 8 29 11 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/5aff0179-25fa-4224-9811-5f55d7a76d12">
<img width="667" alt="Screen Shot 2023-05-27 at 8 29 39 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/9a6d7f93-7a33-4621-8c21-462bd6fb2b39">
<img width="661" alt="Screen Shot 2023-05-27 at 8 29 49 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/9af8278c-b1e9-41b5-91b4-03fbbfabd674">
<img width="446" alt="Screen Shot 2023-05-27 at 8 30 17 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/cf8ebcb1-1df8-48be-86fd-ffee44c70eda">
<img width="912" alt="Screen Shot 2023-05-27 at 8 34 15 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/09ed9bbe-e565-4198-b2cf-b22f30f34a61">
<img width="581" alt="Screen Shot 2023-05-27 at 8 30 47 AM" src="https://github.com/ExistentialAudio/BlackHole/assets/134442108/3a5f7d17-db05-422c-a702-fb0725c0d40a">

**Comment by devinroth** [comment]
There’s a bug with multi-outputs.

https://github.com/ExistentialAudio/BlackHole/wiki/Multi-Output-Device#4-select-output-devices

Probably easier to just stick with 2ch.

Devin

**Comment by bangagurlmusic** [comment]
Okay, thanks Sent from my iPhoneOn May 27, 2023, at 10:15 AM, Devin Roth ***@***.***> wrote:﻿
There’s a bug with multi-outputs.

https://github.com/ExistentialAudio/BlackHole/wiki/Multi-Output-Device#4-select-output-devices

Probably easier to just stick with 2ch.

Devin

**Comment by bangagurlmusic** [comment]
bummer...thanks for your help!

**Comment by devinroth** [comment]
What exactly are you trying to achieve with the 16ch version?

Devin
